### ISSUE TITLE
[Accessibility] No focus when visiting hfs

### ISSUE THREAD

**Issue by Timtam** [rejetto/hfs#593]
Right now, when using HFS 0.52.7 with a screen reader like the freely available NVDA, visiting the page will cause the web interface to not get auto-focused. Instead you'll have to press tab or re-focus the window once by tabbing out and back in. That is kinda abnormal behaviour as that isn't what happens with any other webpage.

**To Reproduce**
1. Launch NVDA
2. Go to 'https://hoard.reaperaccessibility.com
3. Notice how you don't get any speech feedback although the webpage has loaded.

**Describe the bug**

I don't know what exactly causes this issue, but its a little bit annoying as we'll have to deal with alot of people who might not know how to properly handle this behaviour. As described above, the auto-focus doesn't seem to work well. Maybe focus should be moved manually after finishing the web page load to the very first element of the UI, e.g. the navigation or something?

**Environment (please complete the following information):**
 - OS: Linux
 - HFS Version 0.52.7
 - Browser Chrome (it seems to work on Firefox)
 - Screen Reader NVDA (latest alpha)

**Comment by rejetto** [comment]
I use a mac and make my tests with VoiceOver, I cannot install nvda.
But i noticed that at start the focus is on a hidden element, that may be the cause.
I made a fix. Can you give me a feedback to my solution?
0.53.0-alpha2.2 [hfs-linux.zip](https://github.com/rejetto/hfs/files/15314083/hfs-linux.zip)

not sure if you are using binary or npx

**Comment by Timtam** [comment]
I'm running via npx since i'm running on ARM Linux. Could you publish it via npx so that I can give it a try? Thanks!

**Comment by rejetto** [comment]
sure, you can get it with npx hfs@exp

**Comment by Timtam** [comment]
Thanks. Yep, that seems to have fixed the error. Thanks alot!

**Comment by rejetto** [comment]
thanks for your feedback!

**Comment by rejetto** [comment]
fixed in 0.53.0-alpha3
thanks
